jcl/src/java.base/share/classes/openj9/internal/tools/attach/target/DiagnosticUtils.java

```diff
@@ -472,6 +919,32 @@ private static DiagnosticProperties doCheckpointJVM(String diagnosticCommand) {
 			+ "          agentLibrary: the absolute path of the agent%n"
 			+ "          agent option: (Optional) the agent option string%n";
 
+/*[IF JFR_SUPPORT]*/
+/*[IF JAVA_SPEC_VERSION == 11]*/
+	private static final String DIAGNOSTICS_JFR_START_HELP = "Starts a new Recording%n%n"
+			+ JFR_FORMAT_PREFIX + DIAGNOSTICS_JFR_START + JFR_START_OPTION_HELP;
```

Comment:
The duplicated code can be reduced with JPP structure like following:
```
/*[IF JFR_SUPPORT &  (JAVA_SPEC_VERSION >= 11)]*/
	private static final String DIAGNOSTICS_JFR_START_HELP = "Starts a new Recording%n%n"
/*[IF JAVA_SPEC_VERSION == 11]*/
		+ JFR_FORMAT_PREFIX + DIAGNOSTICS_JFR_START
/*[ENDIF] JAVA_SPEC_VERSION == 11 */
		+ JFR_START_OPTION_HELP
/*[ENDIF] JFR_SUPPORT &  (JAVA_SPEC_VERSION >= 11) */
```
This assumes JDK 17+ share same string.